### ISSUE TITLE
Inconsistent slowdown (5ms - 2secs) on endpoints of central-ledger

### ISSUE THREAD

**Issue by yosheeck** [mojaloop/project#1152]
**Summary**:
When running central-ledger locally, I observed random, hard to justify delays for even simplest endpoints.

**Severity**:
High

**Priority**:
Medium

**Expected Behavior**
Endpoints like /health and /enums which effectively "just return JSON" (meaning - those don't do much behind of real processing) should have consistent and quite small RTT (round trip time).

**Steps to Reproduce**
  1. Run central ledger locally (with docker-compose-ed dependencies e.g. kafka)
  2. Hit 127.0.0.1:3001/enums endpoint many times with **MANUAL tool** (browser, postman etc.) or with significant random delays if using any auto-script. It doesn't reproduce when using burst-type tools (like curl in a loop or ab).
  3. The reply return time is random, generally in range 2ms - 2sec with linear distribution.

**Specifications**
  - Component (if known): anything with node.js + kafka (https://github.com/Blizzard/node-rdkafka), but found on central-ledger
  - Version: 8.7
  - Platform: any
  - Type of testing: manual performance
  - Bug found/raised by: @yosheeck 

**Reproduction**:
  - the issue has been reproduced/confirmed locally by @oderayi
  - to clearly and visibly reproduce the issue on central ledger the easiest way is to change the consumeTimeout to some big number for all consumers. For example by changing code in node_modules/@mojaloop/central-services-stream/src/kafka/consumer.js in Consumer::constructor() to 5000, by adding this line after config initiation:
    config.options.consumeTimeout = 5000

**Reason**:
**TLDR:** Kafka allocates libuv threads for consuming, so other I/O (including network replies) is blocked.
**Reason details:**
Each node-kafka consumer when called by consume() allocates one thread from libuv to wait until timeout (default is 1s). Node.js (by libuv) by default creates a pool of 4 threads for UV. If the number of kafka consumers is >3, they may block all UV threads for a timeout time, effectively blocking most of I/O. When that happens, hapi/node stack needs to wait for a free thread to send any data to the user - making all replies slow.
As locally running central ledger **starts 8 consumers,**, they effectively compete for a free thread by up to 2 seconds (4 threads in the pool, 8 consumers, each waits up to 1s on a thread).
On prod/perf there is potentially less consumers turned on per one central-ledger instance, so impact on producation is not yet determined.

**Proposed Solution**:
Set UV_THREADPOOL_SIZE env variable to significantly bigger number than default (4), by one line in index.js:
    process.env.UV_THREADPOOL_SIZE = 12
This will start more libuv threads for the pool.

One of the ways 3rd party libs approach it is the one I found in our dependencies (OracleDB driver of knex): https://github.com/knex/knex/blob/master/lib/dialects/oracledb/index.js - in first 20 lines it changes the UV_THREADPOOL_SIZE.

**Impact**:
Because of the differences in setup, this issue normally impacts only local development and generally shouldn't be visible on "production deployed" environment, as typical deployment spreads the kafka consumers over several instances, so the UV_THREADPOOL_SIZE limit is normally never hit. This was confirmed as part of https://github.com/mojaloop/project/issues/1141
However for local development and performance testing it was a blocking issue.

**Solution introduced**:
In starting central-ledger code we introduced 12 threads allocation.
(The code of process.env.UV_THREADPOOL_SIZE = 12 in https://github.com/mojaloop/central-ledger/blob/3d407d0d2b0410d513df1d3ad30ab111bd91619a/src/api/index.js#L3)

**Additional reading/resources:**
  - issue report on node-rdkafka https://github.com/Blizzard/node-rdkafka/issues/363
  - some other random project having similar issue: https://github.com/nodemailer/nodemailer/issues/963
  - thread pool explanation: http://docs.libuv.org/en/v1.x/threadpool.html

**Notes**:
  - central-ledger has been confirmed to be affected, but potentially there is more components to be verify this behaviour against
  - Severity when opened: High
  - Priority when opened: Medium

  - there is potentially known quick fix for it - now being tested
